### PR TITLE
[D1] db.prepare multiple queries in a single statement

### PR DESCRIPTION
**Pull request by Oxyjun**
### Summary

Adding a short section on adding multiple queries in a single statement.

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Review by marciocloudflare — APPROVED** [review]

File: src/content/docs/d1/build-with-d1/d1-client-api.mdx [hunk 1]
```diff
@@ -23,6 +23,21 @@ However, if you still choose to use a static statement you can use the following
 const stmt = db.prepare('SELECT * FROM users WHERE name = "John Doe"');
 ```
 
+You can pass multiple queries through `db.prepare()`.
+
+### Pass multiple queries in a single statement
```
Comment (vy-ton) [inline src/content/docs/d1/build-with-d1/d1-client-api.mdx:28]: Section is too high up for a less common use case. Can defer fixing this when we create the API section.

I would expect this closer to `Reuse prepared statements` section to consolidate more specific reference related to  prepared statements.

Comment (vy-ton) [inline src/content/docs/d1/build-with-d1/d1-client-api.mdx:28]: Let's create a changelog entry as well, helpful to have users aware of API changes even additive ones

Comment (Oxyjun) [reply src/content/docs/d1/build-with-d1/d1-client-api.mdx:28]: Understood - will reserve this change as part of the new API docs restructure

Comment (Oxyjun) [reply src/content/docs/d1/build-with-d1/d1-client-api.mdx:28]: Understood - I'll add it to the new [API cleanup PR](https://github.com/cloudflare/cloudflare-docs/pull/17923) (i.e. at the same time as the corresponding documentation goes live).

**Comment by Oxyjun** [comment]
This PR is superseded by https://github.com/cloudflare/cloudflare-docs/pull/17923

**Pull request closed without merging**